### DISCUSSION TITLE
Label GPG keys

### DISCUSSION THREAD

**Discussion by ferferga** [community/community#6252] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello!

I have a few GPG keys added to my account. However, there's no clear way to check which one is which using GH's UI (you must check manually the key ID and compare it to the GPG key installed in your devices).

It would be great if, like with the API Access Token, we could name/label those for easier recognition.

**Comment by cdepillabout** [comment, marked as answer]
It appears that this feature has been implemented.

I just added a new GPG to my account.  There is now an input box for a GPG key title, and that is shown on the page with the list of keys.  It now is similar to SSH keys.

**Reply by ferferga** [reply]
I see the option as well, but we can't edit already existing GPG labels.
